### PR TITLE
[Orders] add disabled_order_auto_synchronization

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/personal_data/state.py [hunk 1]
```diff
@@ -236,6 +239,9 @@ async def on_refresh_successful(self):
         """
         raise NotImplementedError("on_refresh_successful not implemented")
 
+    def _is_synchronization_enabled(self):
+        return True
```
Comment (Herklos) [inline octobot_trading/personal_data/state.py:243]: is it overwritten somewhere?

File: octobot_trading/util/test_tools/exchanges_test_tools.py [hunk 2]
```diff
@@ -245,5 +253,7 @@ async def wait_for_other_status(order: personal_data.Order, timeout) -> personal
             return personal_data.create_order_instance_from_raw(
                 order.trader, raw_order, force_open_or_pending_creation=False
             )
+        if time.time() - t0 + constants.CREATED_ORDER_FORCED_UPDATE_PERIOD >= timeout:
+            break
```
Comment (Herklos) [inline octobot_trading/util/test_tools/exchanges_test_tools.py:257]: should we add a debug log here or it's not necessary?

File: octobot_trading/personal_data/state.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_trading/personal_data/state.py:243]: yes, in OrderState

File: octobot_trading/util/test_tools/exchanges_test_tools.py (hunk 2 above)
Comment (GuillaumeDSM) [reply octobot_trading/util/test_tools/exchanges_test_tools.py:257]: No we don't this check is just to kip an asyncio.sleep when we know the next while condition will be False (and therefore we sleep for nothing)

File: octobot_trading/personal_data/state.py (hunk 1 above)
Comment (Herklos) [reply octobot_trading/personal_data/state.py:243]: Right, my bad!

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**